fili-core/src/main/java/com/yahoo/bard/webservice/data/metric/protocol/protocols/TimeAverageMetricTransformer.java

```diff
@@ -42,17 +42,21 @@
         return TimeAverageMetricMakerConfig.timeMakerConfigs.keySet();
     }
 
+    private MetricTransformer delegate;
+
     /**
      * Constructor.
      *
      * Left protected to allow subclasses to add custom time intervals for reaggregation.
      *
      * @param protocolSupportSupplier  A source for the default protocol support used by the makers.
      * @param  makerConfigMap  A collection of maker configurations to use when making time reaggregations.
+     * @param  delegate  A metric transformer to send to if no matching core value is found in this one.ß
      */
     protected TimeAverageMetricTransformer(
             Supplier<ProtocolSupport> protocolSupportSupplier,
-            Map<String, TimeAverageMetricMakerConfig> makerConfigMap
+            Map<String, TimeAverageMetricMakerConfig> makerConfigMap,
+            MetricTransformer delegate
```

Comment:
We don't seem to be assigning the delegate to a field or anything...

Reply:
addressed now